### PR TITLE
New Education Background 

### PR DESCRIPTION
**Pull request by LuckyHariharan**
EDIT:  Added tests using Jest and React testing framework to ensure drop down menu works as expected. 
- `npm run test` would not work unless I changed "test": "env TZ='UTC' jest", to   "test": "set TZ=UTC && jest", in the package.json
- npm install -g win-node-env FIXED IT FOR ME
 
https://www.loom.com/share/63b0d43d27b746c4bd87865f1b29fa58

Hi Vithushan, I created the new Education Background page for the Skillify career quiz. https://www.loom.com/share/a1ac476414a04e51aeb779c4eeecaa68

### CODE REVIEW

**Comment by vithushan19** [comment]
This looks good but I think you should create a test using react testing library

https://testing-library.com/docs/react-testing-library/intro/

@kwiedenhaupt 
I think you could do something similar for your feature

**Comment by LuckyHariharan** [comment]
> This looks good but I think you should create a test using react testing library
> 
> https://testing-library.com/docs/react-testing-library/intro/
> 
> @kwiedenhaupt 
> I think you could do something similar for your feature

Sounds good! Ty for the resource I'll look into it now.

**Comment by LuckyHariharan** [comment]
@vithushan19 ready for re-review, added testing in my latest commits. https://www.loom.com/share/63b0d43d27b746c4bd87865f1b29fa58

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "lint": "next lint",
     "start": "next start",
-    "test": "env TZ='UTC' jest",
+    "test": "set TZ=UTC && jest",
```
Comment (vithushan19) [inline package.json:10]: what does this change do? we might have to update things on vercel to make it work on production

File: components/quizzes/careerQuiz/EduBackground.tsx [hunk 2]
```diff
@@ -1,45 +1,123 @@
...
+                <input
+                  type="text"
+                  name="degree"
+                  value={degree}
+                  onChange={(e) => setDegree(e.target.value)}
+                  className="shadow appearance-none border border-gray-500 rounded-lg px-10"
+                />
+              </div>
+            )}
+          {showExperienceDialog && (
```
Comment (vithushan19) [inline components/quizzes/careerQuiz/EduBackground.tsx:111]: why is this called a dialog?

File: components/quizzes/careerQuiz/EduBackground.test.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import "@testing-library/jest-dom/extend-expect";
+import { fireEvent, render, screen } from "@testing-library/react";
+import React from "react";
+import EduBackground, { EducationLevel } from "./EduBackground";
+
+describe("EducationBackground component", () => {
```
Comment (vithushan19) [inline components/quizzes/careerQuiz/EduBackground.test.tsx:6]: this is awesome! great work getting up to speed on this testing library.

File: package.json (hunk 1 above)
Comment (LuckyHariharan) [reply package.json]: I was missing the env dependencies. npm install -g win-node-env fixed this for me. undid this change.

File: components/quizzes/careerQuiz/EduBackground.tsx (hunk 2 above)
Comment (LuckyHariharan) [reply components/quizzes/careerQuiz/EduBackground.tsx:111]: I misunderstood what dialog meant. changed to showExperienceInput

**Comment by LuckyHariharan** [comment]
> this is looking great to me! nice work! I noticed the PR is still a draft. Is there anything else you're looking to add before merging this in? It looks pretty good to me.

Nothing left! Ready for your review!

**Pull request merged**